bundlewrap/utils/__init__.py

```diff
@@ -5,6 +5,7 @@
 from inspect import isgenerator
 from os import chmod, close, makedirs, remove
 from os.path import dirname, exists
+from passlib.hash import apr_md5_crypt
```

Comment:
```suggestion
```

Reply:
Please move that down to where requests is imported from passlib.hash import apr_md5_crypt